planning/behavior_path_avoidance_module/include/behavior_path_avoidance_module/helper.hpp

```diff
@@ -89,6 +89,7 @@ class AvoidanceHelper
   double getAvoidanceEgoSpeed() const
   {
     const auto & values = parameters_->velocity_map;
+
```

Comment:
unnecessary change...?

Reply:
fixed in https://github.com/autowarefoundation/autoware.universe/pull/5869/commits/e30a50e04751ccbafaca49cc593609877654cfaa